apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/src/components/site-icon/index.tsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useSelect } from '@wordpress/data';
+import { Icon, wordpress } from '@wordpress/icons';
+import { __ } from '@wordpress/i18n';
+
+export default function SiteIcon(): JSX.Element {
+	const currentSiteId: number = window._currentSiteId;
+
+	// The site icon url should be available from /wp/v2/settings or similar in the future,
+	// but for now it's accessible at the index endpoint.
+	// https://github.com/WordPress/gutenberg/blob/68c3978be352c6d3982f73bcf8c80744608c53c8/lib/init.php#L160
+	const siteIconUrl = useSelect(
```

Comment:
Currently, `siteIconUrl` is of type `any`

```suggestion
	const siteIconUrl: string | undefined = useSelect(
```

Reply:
Updated, thanks.